.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,12 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Report a data error
+    url: https://www.wikidata.org/w/index.php?action=edit&title=Wikidata:WikiProject_Govdirectory/Error_reports&section=new&preloadtitle=Rep%20ort+of+data+error&preload=Wikidata%3AWikiProject_Govdirectory%2FError_reports%2FError_report_template
```

Comment:
I would suggest changing "preloadtitle=Rep%20ort+of+data+error" to "preloadtitle=Report+a+data+error".

Reply:
preloadtitle has been changed